types/scene/wlr_scene.c

```diff
@@ -1872,23 +1884,18 @@ static void scene_entry_render(struct render_list_entry *entry, const struct ren
 				.use_optimized_blur = scene_rect->backdrop_blur_optimized,
 				.blur_data = &scene->blur_data,
 				.ignore_transparent = false,
+				.clipped_region = {
```

Comment:
Also, don't forget to initialize the `wlr_render_texture_options` clipped_region to `{0}` everywhere.

Reply:
does `{0}` "just" initialize all the fields to zero? I don't have to define every field?

Reply:
Yes. It's a lot easier :)

Reply:
Wait do you mean `fx_render_texture_options`? because I didn't change `wlr_render_texture_options`